spec/models/comment_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  subject do
+    user = User.create(name: 'Graha Tia', photo: '', bio: 'Crystal exharc')
+    post = Post.new(author: user, title: 'Stand tall my friends', text: 'Our journey will never end')
+    Comment.new(author: user, post:, text: 'May those who walked before, lead those who walk after')
+  end
+
+  before { subject.save }
+
```

Comment:
- [ ] You have some errors on your test files, please fix them for the next step.


>   /Users/rustamjon/Rustamxon7/blog-appssss/spec/models/comment_spec.rb:7: syntax error, unexpected ','
>   ...omment.new(author: user, post:, text: 'May those who walked ...